knn-classifier/README.md

```diff
@@ -23,7 +23,7 @@ You can see example code [here](https://github.com/tensorflow/tfjs-models/tree/m
     <!-- Load MobileNet -->
     <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/mobilenet@0.1.1"></script>
     <!-- Load KNN Classifier -->
-    <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/knn-classifier@0.1.0"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/knn-classifier@latest"></script>
```

Comment:
you can remove @latest entirely if you want

Reply:
Oops. Didn't see this comment until the PR is merged. I think it should be equivalent in this case, except the slightly longer  but more explicit URL.